index.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap"
+    rel="stylesheet" />
+  <title>KMarketer</title>
+</head>
+<body>
+  <div class="header">
+      <h2 class="header__logo">KMarketer</h2>
+      <div class="header__nav-left">
+        <div class="nav__items">
+          <a class="nav__item" href="">home</a>
+          <a class="nav__item" href="" alt="">product</a>
+          <a class="nav__item" href="" alt="">pricing</a>
+          <a class="nav__item" href="" alt="">contact</a>
+          <img src="./image/Vector.svg" alt="">
```

Comment:
These images must have a class-name. In BEM, it is recommended that every tag must have a class name